_pages/about.md

```diff
@@ -11,19 +11,29 @@ news: false # includes a list of news items. TODO: set to true when needed
+### Introduction
 
-## Multimodal Time Series Models
-Most time-series models in the literature are restricted to only handle numerical time-series data. However, numerical data often only paints a partial picture of the state of a system of interest. In real-world settings, multiple modalities are often available as input, and time series analysis or prediction tasks could potentially be improved by appropriately taking them into account. In this segment of the workshop, we seek to discuss the challenges involved in building such models, and welcome contributions including, but not limited to, attempting to build multimodal time series models ([15](https://arxiv.org/abs/2305.16556)), contributions comparing various approaches to fuse different modalities, datasets where time series modalities are paired with data from other modalities (such as text or images). In a similar vein of building multimodal time series models, we welcome contributions that build text-conditioned forecasting models ([16](https://arxiv.org/abs/2405.13522)), where textual cues such as news articles or event-related data can be utilized, in addition to the time series data available in a dataset, to condition a model to produce better predictions. We further invite contributions that explore the ability of such multimodal models to understand or infer semantic (such as causal) information between variables for time series analysis.
+Foundation models have significantly changed the approach to building machine learning models in areas such as natural language processing, where models are pretrained on large amounts of diverse data and then adapted for downstreams tasks, often in a _zero-shot_ fashion.
+Such pretrained models have started to gain traction in the time series community and several recent works have developed and open-sourced foundation models for time series tasks, particularly forecasting.
+These works have opened up new research directions and challenges.
+This workshop seeks to provide a forum for researchers and practitioners to understand the progress so far and push the frontier of research on time series foundation models (TSFMs). 
```

Comment:
The introduction seems very focused on time series foundation models, which I'd personally consider as only 1 of t he 4 major topics of the workshops. I'd expect the introduction to refer to general progress in the development/usage of **large models** for time series (which subsumes TSFMs), which includes also LLMs for time series, and in general even large (non-pretrained) transformers for time series. 
i.e. large models for time series

I'd suggest keeping the same text, but also adding information about the other 3 topics that's currently in the about page. And ending with a note that this workshop aims to frontier of research on time series in /the age of large models/, in all these directions.

Reply:
Massaged the introduction a bit to also highlight other developments. Let me know what you think.

Reply:
Looks good, thanks!